Initial commit. Create retrieve_environment.py.

seigodev opened the pull request:
issue: Search mizar dependency.  #21
mizarファイルから環境部を取り出すプログラムの作成をしました。
make_library_dependence(): ライブラリの依存関係を示す辞書の作成（初期化）を行う。mizarライブラリファイルが格納されているディレクトリへアクセスし、.mizファイルを取得する。

seigodev committed:
Create retrieve_environment.py.

issue: Search mizar dependency.  #21
mizarファイルから環境部を取り出すプログラムの作成
make_library_dependence(): 辞書miz_files_dictを作成する。mizarライブラリファイルが格納されているディレクトリへアクセスし、.mizファイルを取得する。

aabaa commented on retrieve_environment.py:
```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+
+def make_library_dependence():
```
dependence -> dependency

aabaa commented on retrieve_environment.py:
```diff
@@ -0,0 +1,35 @@
+    os.chdir(path)
+    miz_files = glob.glob("*.miz")  # mmlディレクトリの.mizファイルを取り出す
+
+    os.chdir("../")
```
```python
cwd = os.getcwd()
try:
    os.chdir(path)
    # ここで何らかの操作
finally:
    os.chdir(cwd)
```

aabaa commented on retrieve_environment.py:
```diff
@@ -0,0 +1,35 @@
+    for category in categories:
+        miz_files_dict[category] = dict()
+
+    path = Path("mml")
```
ベタ書きしないでパスを管理するためのファイルを別途用意して管理する

seigodev committed:
Fix requested changes.

・dependence -> dependency (6行)
・try finallyで処理(30-36行)
・パスは1つだけなので、とりあえず上部に書きました。
